app/src/main/java/com/example/airlinesapp/ui/home/passengers/PassengersViewModel.kt

```diff
@@ -0,0 +1,48 @@
+package com.example.airlinesapp.ui.home.passengers
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.paging.PagingData
+import com.example.airlinesapp.models.Passenger
+import com.example.airlinesapp.util.Constants.LOADED
+import com.example.airlinesapp.util.Constants.LOADING
+import com.example.airlinesapp.util.Constants.CHECK_NETWORK_ERROR
+import io.reactivex.disposables.CompositeDisposable
+import javax.inject.Inject
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.Transformations
+import com.example.airlinesapp.di.network.ApiRepository
+
+
+class PassengersViewModel @Inject constructor(private val repository: ApiRepository) :
+    ViewModel() {
+
+    private var networkState: MutableLiveData<String> = MutableLiveData(LOADING)
```

Comment:
get rid of it. not needed. 

you can use a live data with boolean to indicate loading and another for is empty ... etc